### DISCUSSION TITLE
BUG - Can't pass "action" to onActiveStartDateChange method

### DISCUSSION THREAD

**Discussion by laocker96** [wojtekmaj/react-calendar#653] [Q&A] [answered]
Version `"react-calendar": "^3.9.0",`

I'm using the example from documentation as a test, but parameter `action` is not defined in the function `onActiveStartDateChange`.

This is a code example and what i have as error:

![Cattura3](https://user-images.githubusercontent.com/32465510/196684863-18b4dd01-cd32-4086-b1ae-c91e8a29e521.PNG)

As you can see prop `action` is missing:

![Cattura1](https://user-images.githubusercontent.com/32465510/196685026-d347d983-3637-40dc-91c7-eea7443bac1f.PNG)

Anyone can help?

**Comment by wojtekmaj** [comment, marked as answer]
Hey! Please raise this issue to DefinitelyTyped, an external repo providing types for React-Calendar. Thanks!
